packages/dx-react-scheduler/src/plugins/view-switcher.tsx

```diff
@@ -17,6 +18,9 @@ class ViewSwitcherBase extends React.PureComponent<ViewSwitcherProps> {
   static components: PluginComponents = {
     switcherComponent: 'Switcher',
   };
+
+  getMemoCurrentView = memoize((name, type, displayName) => ({ name, type, displayName }));
```

Comment:
We should extract this code outside the class.

Also, can we extract new global constants and function into `scheduler-core` package?